src/DataTypes/Course.jl

```diff
@@ -41,10 +41,13 @@ mutable struct Course
     metrics::Dict{String, Any}          # Course-related metrics
     metadata::Dict{String, Any}         # Course-related metadata
 
+    termReq::Int                        # Number of terms that must be completed before enrolling
+    passrate::Float64                   # Percentage of students that pass the course
```

Comment:
I haven't seen the notion of a minimum number of terms enrolled prior to enrolling in a specific course. What was the reason to add this? Is there a program that required this or perhaps was it implemented in the simulation model this was based off of?

pastorate looks good and I think should be implemented in this package no matter what we decide to do.

One note sort of nitpicky note, but all the course fields are snake cased, so these should be `term_req` and `pass_rate` 🤠

Reply:
Yup, termReq is redundant here, it should store in the metadata, it's removed from Course struct.